tests/pytests/test_profile.py

```diff
@@ -87,10 +87,10 @@ def testProfileSearch(env):
                           ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1L, 'Size', 1L]]]]]]
   env.assertEqual(actual_res[1][3], expected_res)
 
-  if not check_server_version(env, '6.00.20'):
+  if not check_server_version(env, '6.2.0'):
```

Comment:
Maybe we need to check version > '6.0'? We can use `semantic_version` for the comparison.

Reply:
the fix for reply depth was introduced in 6.2.0. The previous version was wrong